doc/lsst.verify/reviewing-verification-json-on-the-command-line.rst

```diff
@@ -0,0 +1,76 @@
+.. _lsst.verify.inspect_job:
+
+###########################################################################
+Reviewing verification JSON outputs on the command line with inspect_job.py
+###########################################################################
+
+|inspect_job| is a command-line tool that lets you quickly see what information is stored in the ``*verify.json`` files generated whenever you run code that integrates with LSST's verification framework (:doc:`lsst.verify <index>`).
+It's particularly useful when you want to see measurement values and don't need a full report comparing measurements to metric specifications.
+
+.. seealso::
+
+   See :doc:`scripts/inspect_job.py` for a complete reference of that script's command-line interface.
+
+.. _SQR-019: https://sqr-019.lsst.io
+
+Running inspect_job.py
+======================
+
+|inspect_job| takes one or more verification framework JSON files as command-line arguments.
+Since JSON files output by the verification framework generally have names that end with :file:`verify.json`, you can quickly inspect all available outputs by using a ``*.verify.json`` wildcard:
```

Comment:
Above you say `*verify.json`.  That seems safer than `*.verify.json`, but maybe I'm not well informed on how these files are generated.